test/e2e/tests/network/add-custom-network.spec.js

```diff
@@ -958,19 +949,10 @@ async function checkThatSafeChainsListValidationToggleIsOn(driver) {
   await driver.waitForSelector(securityAndPrivacyTabRawLocator);
   await driver.clickElement(securityAndPrivacyTabRawLocator);
 
-  const useSafeChainsListValidationToggleSelector =
-    '[data-testid="useSafeChainsListValidation"]';
-  const useSafeChainsListValidationToggleElement = await driver.waitForSelector(
-    useSafeChainsListValidationToggleSelector,
-  );
-  const useSafeChainsListValidationToggleState =
-    await useSafeChainsListValidationToggleElement.getText();
-
-  assert.equal(
-    useSafeChainsListValidationToggleState,
-    'ON',
-    'Safe chains list validation toggle is off',
-  );
+  await driver.findElement({
+    xpath:
+      "//div[@data-testid='useSafeChainsListValidation']//label[contains(@class, 'toggle-button') and contains(@class, 'toggle-button--on')]",
+  });
```

Comment:
we need to use an xpath here since there's a long list of toggles, without a way of differentiating to which feature belong to

Reply:
Nice job!!